weave-js/src/entrypoint.tsx

```diff
@@ -10,6 +10,17 @@ import {WeaveMessage} from './components/Panel2/WeaveMessage';
 import {NotebookComputeGraphContextProvider} from './contextProviders';
 import {URL_LOCAL, URL_RECENT, URL_WANDB} from './urls';
 
+// These get popuated via /__frontend/env.js and are defined in weave_server.py
+declare global {
+  interface Window {
+    CONFIG: {
```

Comment:
should we namespace this with something relating to `wandb` or `weave`?

Reply:
Yeah that's a very good idea, this will stomp over the wandb config.  I think it might be ok because going to anything rendering wandb will load it's index.html and stomp over it...